packages/esm-appointments-app/src/appointments/scheduled/scheduled-appointments.component.tsx

```diff
@@ -43,7 +43,7 @@ const ScheduledAppointments: React.FC<ScheduledAppointmentsProps> = ({ visits, a
   );
 
   useEffect(() => {
-    const dayjsDate = dayjs(date);
+    const dayjsDate = dayjs(selectedDate);
```

Comment:
Again, not germane to this PR, but since `const now` is only recalculated when `selectedDate` changes, this could appear to give stale results, especially in 24/7 operations.

Reply:
Yeah, I think this is probably fine. Tbh, I don't expect a 24/7 facility to use appointments regularly outside of business hours.